<launch>
  <rosparam command="load" file="$(find cart2)/data/config/manager.yaml" />
  <node pkg="cart2" type="hlmanager" name="hl_manager" machine="controlled">
      <param name="LocalFixSim" value="true" />     
      <remap from="ref_point" to="LFPoint" />
      <remap from="ref_track" to="TrackPoint" />
  </node>
</launch>
